code/style.css

```diff
@@ -1,4 +1,305 @@
-html {
-  background: #0025ff;
-  color: #fff;
+@import url('https://fonts.googleapis.com/css2?family=Livvic&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    background-color:#fcfcfc;
+}
+
+p {
+    font-size: 12px;
+}
+
+
+
+/***** HEADER ******/
+.header {
+    background-color: #1b1b1b;
+    color:#eeedea;
+    font-family: 'Livvic', sans-serif;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 80px;
+}
+
+.logo { 
+    display: flex;
+    width: 80px;
+    padding-left: 10px;
+   }
+
+.header-title {
+    margin: 0%;
+    font-size: 16px;
+    padding: 0 10px;
+}
+
+.header-title {
+    display: none;
+}
+
+.mobile-hamburger { 
+    display: flex;
+    width: 20px;
+    height: 20px;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-right: 20px;
+}
+
+.mobile-hamburger div {
+    width: 25px;
+    height: 2px;
+    background: #eeedea;
+}
+
+.navigation {
+    display: none;
+}
+
+.navigation a { 
+    text-decoration: none;
+    color: white;
+    margin: 0px 15px;
+}
+
+.mobile-title {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 20px 0;
+    text-align: center;
+    font-family: 'Livvic', sans-serif;
+    font-weight: 700;
+    font-size: 24px;
+}
+
+
+
+/************** BIG NEWS SECTION ******************/
+ .big-news {
+     background-color: rgb(248, 248, 248);
+     width: 100%;
+ }
+
+ .bn-card {
+     display: flex;
+     flex-direction: column;
+     justify-items: center;
+     align-items: center;
+     margin: 20px 20px;
+     text-align: center;
+     padding: 10px 10px;
+     background-color: rgb(253, 253, 253);
+     color: rgb(53, 52, 52);
+ }
+
+ .img-container {
+     overflow: hidden;
+     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 
+                0 6px 20px 0 rgba(0, 0, 0, 0.19)
+ }
+
+ .img-container img {
+     height: 100%;
+     width: 100%;
+     object-fit: cover;
```

Comment:
This is the best property there is in my opinion when working with different sized images 👍